src/client/src/apps/SimpleLauncher/spotlight/components/getInlineSuggestionsComponent.tsx

```diff
@@ -1,33 +1,32 @@
 import React from 'react'
-import { DetectIntentResponse } from 'dialogflow';
-import { Platform } from 'rt-platforms';
+import { DetectIntentResponse } from 'dialogflow'
+import { Platform } from 'rt-platforms'
 import {
   getCurrency,
   getCurrencyPair,
   getNumber,
   handleIntent,
   isSpotQuoteIntent,
   isTradeIntent,
-  mapIntent
-} from '../intents';
-import { BlotterFilters, DEALT_CURRENCY, SYMBOL } from '../../../MainRoute/widgets/blotter';
-import { Intent, Suggestion } from './styles';
-import { InlineBlotter } from './InlineBlotter';
-import { InlineQuote } from './InlineQuote';
+  mapIntent,
+} from 'rt-intents'
```

Comment:
Nice!

Reply:
As mentioned on Slack, I will rename this `rt-interop` ?